lifecycle-operator/config/rbac/role.yaml

```diff
@@ -306,13 +306,13 @@ rules:
 - apiGroups:
   - lifecycle.keptn.sh
   resources:
-  - keptnworkloadinstances/finalizers
+  - keptnworkloadVersions/finalizers
   verbs:
   - update
 - apiGroups:
   - lifecycle.keptn.sh
   resources:
-  - keptnworkloadinstances/status
+  - keptnworkloadVersions/status
```

Comment:
I think the 3 instances here and before need to stay on keptnworkloadinstances

Reply:
hmm, why do they need to stay? the operator does not use WorkloadInstances anywhere anymore

Reply:
Sorry, I forgot that these CRDs are created by us. We should then delete these entries in the RBAC since we don't control anymore WorkloadInstances

Reply:
but this is lifecycle-operator-role, if we remove it, the workload controller won't be able to create workloadVersion, or do I miss something ?

Reply:
according to the process in ArgoCD, I created an enhancement proposal https://github.com/argoproj/argo-cd/issues/15922. If this will be accepted, I can follow-up with a contribution to ArgoCD

Reply:
i believe the required roles have been added here, after the kubebuilder command comments have been added to the controllers: https://github.com/keptn/lifecycle-toolkit/pull/2255/files#diff-6eaf3e43673ae6dc491f9151d5dd7fc1346e3ab1779205df8e78f17d5bb31829R346

Reply:
oh sorry my mistake, of course, this can be removed

Reply:
adapted